ldap2pg.master.yml

```diff
@@ -55,24 +55,24 @@ postgres:
 # **ACL**
 #
 acl_dict:
-  defaults:
+  public:
     inspect: |
-      WITH d AS (
+      -- Returns public if no ACLs are defined in this database.
+      WITH
+      acls AS (
           SELECT
               (aclexplode(datacl)).grantee AS grantee,
               (aclexplode(datacl)).privilege_type AS priv
           FROM pg_catalog.pg_database
           WHERE datname = current_database()
+      ),
+      static AS (
+          SELECT NULL AS namespace, 'public' AS rolname
       )
-      SELECT NULL AS namespace, 'public' AS rolname
-      FROM d AS d0
-      LEFT OUTER JOIN d AS d1 ON d1.grantee = d0.grantee AND d1.priv <> d0.priv
-      WHERE d0.grantee = 0 AND d0.priv = 'TEMPORARY' AND d1.priv IS NULL
-      UNION
-      -- CONNECT ACL is not stored on template1 for PUBLIC. So fake it.
-      SELECT NULL AS namespace, 'public' AS rolname
-      WHERE current_database() = 'template1'
-    grant: |
+      SELECT static.* FROM static
+      LEFT OUTER JOIN acls ON acls.grantee = 0
+      WHERE acls.grantee IS NULL;
+    revoke: |
```

Comment:
@cbandy I reword this ACL to use a REVOKE to REVOKE. The inspect looks more complex. The purpose of this ACL is just to revoke public connect which is defaults.